RibirX/Ribir
# Refactor/not depends backend msaa

## Purpose of this Pull Request

1. Remove the anti-aliasing configuration from the painter. The backend will now decide whether to use anti-aliasing. For the wgpu backend, anti-aliasing will be used by default and implemented in the shader, independent of the platform under wgpu.
2. Set the panic hook for all examples to improve panic information visibility in the browser.
3. Update all image test cases to reflect the support for anti-aliasing by default.

## How to Review/Test this Pull Request

You can review the code commit by commit, and run the example to see the result in the browser.

```shell
cargo run-wasm -p counter
## Checklist Before Merging
```

Please ensure the following are completed before merging:
- [x] If this is linked to an issue, include the link in your description.
- [x] If you've made changes to the code or documentation, make sure these are updated in the `CHANGELOG.md` file.
- [x] If you've introduced any break changes, briefly describe them in the `Breaking` section of the `CHANGELOG.md` file.

## Additional Information

**The bot will replace `#pr` in `CHANGELOG.md` with your pull request number. If your branch is out of sync, use `git pull --rebase` to update it.**

If you're unsure about which branch to submit your Pull Request to, or when it will be released after being merged, please refer to our [Release Guide](https://github.com/RibirX/Ribir/blob/master/RELEASE.md).

If you're working on a widget and need help writing test cases, we have some macros that can assist you. Please refer to the [Ribir Dev Helper](https://docs.rs/ribir_dev_helper) documentation.
